schema/routines.sql

```diff
@@ -3883,7 +3883,7 @@ BEGIN
     IF p_id IS NOT NULL AND p_program IS NOT NULL THEN
       SELECT dc.dataCollectionId, app.autoProcProgramId,
         app.processingStatus,
-        concat('[', group_concat(json_object('fileType', appa.fileType, 'fullFilePath', concat(appa.filePath, '/', appa.fileName))), ']') "processingAttachments"
+        concat('[', group_concat(json_object('fileType', appa.fileType, 'fullFilePath', concat(appa.filePath, '/', appa.fileName, 'importanceRank', appa.importanceRank))), ']') "processingAttachments"
```

Comment:
The `importanceRank` field should not be concat'ed in the `fullFilePath`! It should be outside of that. Looks like you got it correctly in the `sp_retrieve_processing_program_attachments_for_dc_group_and_program.sql` file below.

Reply:
Too many parentheses!